pkgs/development/ocaml-modules/arp/default.nix

```diff
@@ -8,50 +8,50 @@
 
 buildDunePackage rec {
   pname = "arp";
-  version = "2.3.1";
+  version = "2.3.2";
 
-  minimumOCamlVersion = "4.06";
+  src = fetchFromGitHub {
```

Comment:
Please revert to the use of `fetchurl` to fetch the *release tarball* instead of the git repository snapshot at the specific tag. The release tarball has version numbers and similar information substituted which the git repository doesn't have.

Reply:
Done.